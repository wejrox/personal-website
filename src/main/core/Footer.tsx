import * as React from "react";
import Row from "reactstrap/lib/Row";
import Col from "reactstrap/lib/Col";

class Footer extends React.Component {

    public render() {
        return (
            <Row>
                <Col>
                    Footer
                </Col>
            </Row>
        );
    }
}

export default Footer;