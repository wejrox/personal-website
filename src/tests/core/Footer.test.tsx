import * as Enzyme from "enzyme";
import * as React from "react";
import Footer from "../../main/core/Footer";

describe("Footer Component", () => {
    const wrapper = Enzyme.shallow(<Footer/>);

    it("renders correctly", () => {
        expect(wrapper).toMatchSnapshot();
    });
});
