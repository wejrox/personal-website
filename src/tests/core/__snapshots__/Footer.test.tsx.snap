// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders correctly 1`] = `ShallowWrapper {}`;
